homeassistant/components/nexia/sensor.py

```diff
@@ -0,0 +1,288 @@
+"""Support for Nexia / Trane XL Thermostats."""
+
+from nexia.const import UNIT_CELSIUS
+
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+    UNIT_PERCENTAGE,
+)
+
+from .const import (
+    ATTRIBUTION,
+    DATA_NEXIA,
+    DOMAIN,
+    MANUFACTURER,
+    NEXIA_DEVICE,
+    UPDATE_COORDINATOR,
+)
+from .entity import NexiaEntity
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up sensors for a Nexia device."""
+
+    nexia_data = hass.data[DOMAIN][config_entry.entry_id][DATA_NEXIA]
+    nexia_home = nexia_data[NEXIA_DEVICE]
+    coordinator = nexia_data[UPDATE_COORDINATOR]
+    entities = []
+
+    ###########################################################################
```

Comment:
We don't use separators like this. A comment is enough.

Reply:
adjusted